app/src/organisms/Desktop/RunProgressMeter/hooks/useRunProgressCopy.tsx

```diff
@@ -127,6 +131,23 @@ export function useRunProgressCopy({
       }
     }
   })()
+  const {reportModuleCommand} = useModuleCommandAnalytics()
+  // Check if error occurred, then determine type
+  let error; 
+  if (typeof runCommandDetails?.data?.error === 'object') {
+    error = runCommandDetails?.data?.error?.errorType;
+  }
+
+  reportModuleCommand({
+    runId: runId!,
```

Comment:
```suggestion
    runId: runId ?? '',
```
Either we coerce it to an empty string, or the interface supports runId being a string | null, and we coerce it to null. Whatever your internal API would prefer working with.